src/styles/app.css

```
@@ -204,3 +204,12 @@
 		background-color: transparent !important;
 	}
 }
+
+@keyframes tourProgress {
```

Stylelint (17.14.0)

[error] Expected keyframe name "tourProgress" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)